### ISSUE TITLE
About the part performance on TigDog in supervied training with HRNet-w32

### ISSUE THREAD

**Issue by yzrs** [JitengMu/Learning-from-Synthetic-Animals#14]
Hi,Thanks for your contribution.
I'm also working on animal pose estimation and do some experiments on TigDog dataset with HRNet-w32 in supervised training.However, I find that my results are quite different from yours on some parts like the shoulder and the hip of the horses, and the hip and the knee of the  tigers,which really confuses me.
![image](https://github.com/JitengMu/Learning-from-Synthetic-Animals/assets/45112462/1eeed9df-cc52-490c-9ac2-eed16928be2d)
I use HRNet-w32 pretrained on the ImageNet as my network.
My data augmentation methods include scaling,rotation,random half body,random horizontal flip and color jitter.
My train dataset and val dataset are from https://github.com/JitengMu/Learning-from-Synthetic-Animals/tree/master/data/real_animal,where I get the train idxs and their corresponding images and labels.Then I convert the labels into coco format for training.
After training, I test my model on the valid idxs from https://github.com/JitengMu/Learning-from-Synthetic-Animals/tree/master/data/real_animal and compute the PCK@0.05 value on these parts.
Here, for horse:
  - eye = left eye[1] + right eye[2]
  - chin = chin[3]
  - shoulder = left shoulder[13] + right shoulder[14]
  - hip = back left hip[17] + back right hip[18]
  - elbow = front left hip[15] + front right hip[16]
  - knee = front left knee[9] + front right knee[10] + back left knee[11] + back right knee[12]
  - hoof = front left hoof[4] + front right hoof[5] + back left hoof[6] + back right hoof[7]
  - 'mean' is weighted average PCK@0.05 value of all parts including tail[8]
![image](https://github.com/JitengMu/Learning-from-Synthetic-Animals/assets/45112462/9f35350c-7c8e-403b-9c3e-104d4993ce95)

The part definition of tiger is similar to horse except that the ankle of tiger corresponds to the horse's knee , and the knee of tiger corresponds to the horse's hip.  
Is there something wrong in my steps? I'm looking forward to your reply

**Comment by yzrs** [comment]
In table, the 'Real' performance is from the paper and the 'experiment' is my results
